FAQ/FAQ-Deployment.md

```diff
@@ -149,6 +150,17 @@ exit
 
 If you need to figure out which environment variable you need to modify, have a look at [this list](https://github.com/btcpayserver/btcpayserver-docker#environment-variables).
 
+### How can I run BTCPay on testnet?
+
+Building on the section above, this is how you configure BTCPay to use `testnet` instead of the default `mainnet`:
+
+```bash
+export NBITCOIN_NETWORK="testnet"
+```
```

Comment:
It's a good idea to add `. btcpay-setup.sh -i` in the next line, otherwise changes won't be applied :)

Reply:
Alright, thanks for the hint – fixed it!